### DISCUSSION TITLE
Version FlorisBoard

### DISCUSSION THREAD

**Discussion by MXC48-zz** [florisboard/florisboard#1289] [Q&A]
Hello, when is the next version of florisboard planned?

**Comment by patrickgold** [comment]
When the development of the preference rework is finished. There's no exact ETA, but it will be mid-Q4/2021 I guess.

You can follow the beta releases on the dev progress if you want more frequent (but less stable) updates, the latest beta release was yesterday.

**Reply by MXC48-zz** [reply]
Thank  you for the response
